package.json

```
@@ -13,22 +13,25 @@
   },
   "license": "BSD-3-Clause",
   "author": "OVH SAS",
+  "files": [
+    "dist"
+  ],
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
   "scripts": {
-    "prebuild": "rm -rf dist/*",
-    "build": "rollup -c",
+    "build": "shx rm -rf dist/* && rollup -c",
```

Here is a small comparaison between `shx` and `rimraf`:

https://packagephobia.now.sh/result?p=shx
- Publish size: **34.2kB**
- Install size: **537kB**


https://packagephobia.now.sh/result?p=rimraf
- Publish size: **15.8kB**
- Install size: **165kB**

[rimraf](https://github.com/isaacs/rimraf) is widely used and seems to be more reliable in my point of view.

In addition you could also use `npx` command to reduce the amount of devDependencies:

```suggestion
    "build": "npx rimraf dist && rollup -c",
```

---

you got me convinced, thanks for pointing that out